db/version_edit.cc

```diff
@@ -55,6 +55,7 @@ enum CustomTag : uint32_t {
   // kMinLogNumberToKeep as part of a CustomTag as a hack. This should be
   // removed when manifest becomes forward-comptabile.
   kMinLogNumberToKeepHack = 3,
+  kIngestFile = 4,
```

Comment:
Can we not change manifest format, instead check whether global seqno exists? For simplicity we can assume `write_global_seqno=false`, if needed.

Reply:
seems yes. I'll take a look